src/ja/worker/message/crashed.py

```diff
@@ -1,31 +1,49 @@
 from typing import Dict
 
+from ja.common.job import JobStatus
 from ja.worker.message.base import WorkerServerCommand
-from ja.common.message.server import ServerResponse
+from ja.common.message.base import Response
 from ja.server.database.database import ServerDatabase
 
 
 class JobCrashedCommand(WorkerServerCommand):
     """
     Informs the server that a job has crashed.
     """
+
     def __init__(self, job_uid: str):
         """!
         @param job_uid: The UID of the job that has crashed.
         """
+        self._job_uid = job_uid
 
     @property
     def job_uid(self) -> str:
         """!
         @return: The UID of the job that has crashed.
         """
+        return self._job_uid
+
+    def execute(self, database: ServerDatabase) -> Response:
+        job_entry = database.find_job_by_id(self._job_uid)
+        if job_entry is None:
+            raise ValueError("There is no job with %s id on the server" % self._job_uid)
+        job_entry.job.status = JobStatus.CRASHED
+        database.update_job(job_entry.job)
+        return Response("Job crashed!", True)
```

Comment:
Please add the job uid to the response message.
That way we can just use the response messages for the server log.